DataDog/integrations-core
# Nils/logs integration beta

### What does this PR do?

Default integration yaml file are updated to take the log section into account.

### Motivation

The agent 6 is now able to collect logs thanks to a specific log section in the yaml files. When there are some defaults log file path for a technology, the default yaml file already contains it so it simplify the log collection.

### Testing Guidelines

Install agent 6 and uncomment the log section for any of the modified integration. Create a file at the path location and start writing in it. You should see your logs coming in your platform.
Do not forget to put `log_enabled: true` in your `datadog.yaml` file.

### Versioning

  for the new section.

### Additional Notes

This is available for Agent version >= 6. You can find more information about logs in the [documentation](https://docs.datadoghq.com/logs/)
